prow/cmd/phaino/local.go

```diff
@@ -185,7 +210,7 @@ func convertToLocal(log *logrus.Entry, pj prowapi.ProwJob, allowPrivilege bool)
 		localArgs = append(localArgs, "-e", env.Name+"="+env.Value)
 	}
 
-	priv, err := checkPrivilege(container, allowPrivilege)
+	priv, err := checkPrivilege(ctx, container, allowPrivilege)
```

Comment:
you should probably check for capabilites as well. kubernetes stopped having support for "allow / dissallow privleged" specifically because this is insufficient.

Reply:
Yeah I'm explicitly ignoring capabilities for the time being. Added a TODO.

I'm intentionally trying to make privileged container execution more annoying than normal in the hopes that this will discourage its use and encourage refactoring the job to not need privilege.